frontend/website-redesign/src/components/Icon.re

```diff
@@ -48,7 +48,7 @@ type kind =
  *  Some icons have not been edited for currentColor properties. TODO is to come back and clean them up when needed.
  */
 [@react.component]
-let make = (~kind, ~size=1.5) => {
+let make = (~kind, ~size=1.5, ~currentColor="white") => {
```

Comment:
i think we decided to remove this prop and just have fill="currentColor" in the svg tag

Reply:
Yep!